source/time_tables/departure_boards.cpp

```diff
@@ -282,17 +282,19 @@ struct JourneyPatternEndsByDirection {
 
 // For the JP of 'tested', check if all the JPP following 'tested' itself are included in the same order
 // in the JP 'direction', starting from  JPP 'direction' itself
-static bool is_jp_end_included_in_direction(const routing::JourneyPatternPoint& tested,
+static bool is_jp_end_included_in_direction(PbCreator& pb_creator,
```

Comment:
```suggestion
static bool is_jp_end_included_in_direction(const vector<StopPoint*>& stop_points,
```
PbCreator isn't necessary as you only need the map of stop points from `pb_creator`.
This seems harmless, but it does create unnecessary coupling and we already have waaaaaayyyyyyy to much of it already in kraken.

Reply:
ok